src/main/java/tools/jackson/databind/MapperFeature.java

```diff
@@ -277,10 +277,11 @@ public enum MapperFeature
      *<p>
      * Note: does <b>not</b> apply to {@link java.util.Map} serialization (since
      * entries are not considered Bean/POJO properties.
+     *
      *<p>
-     * Feature is disabled by default.
+     * Feature is enabled by default.
```

Comment:
Should mention difference between 2.x and 3.0 (there is hopefully an example on this for another feature)

Reply:
@cowtowncoder I actually did remove difference-explanation after refering to other changed deafult settings. This seems like matter of preferences, but I think it would help users a bit.

Reply:
Ok. Well, we do have time to unify handling so I will it as-is for now (with reference to difference).